docs/en-US/component/menu.md

```diff
@@ -80,6 +80,8 @@ menu/popper-offset
 | router                  | whether `vue-router` mode is activated. If true, index will be used as 'path' to activate the route action. Use with `default-active` to set the active item on load. | boolean               | —                     | false    |
 | collapse-transition     | whether to enable the collapse transition                                                                                                                             | boolean               | —                     | true     |
 | popper-effect ^(2.2.26) | Tooltip theme, built-in theme: `dark` / `light` when menu is collapsed                                                                                                | string                | dark / light          | dark     |
+| close-on-click-outside ^(2.5.0) | optional, whether menu is collapsed when clicking outside              | `boolean`             | —               | false                                           |
+| hide-timeout ^(2.5.0) | timeout before hiding a sub-menu when clicking outside          | `number`             | —               | 300                                           |
```

Comment:
I share the same idea as @kooriookami , `hide-timeout` only works in hover mode and I have a other suggestion

add new props `hide-timeout` and 'show-timeout' in menu, Used to set global timeout, `hide-timeout` and 'show-timeout' in sub-menu will cover it. My idea comes from #15051 .
or
remove `hide-timeout` and related functions in menu. we will create a new PR to implement it.

Reply:
Since It is difficult for me to follow up on your idea 1, I decided to remove the `hide-timeout` prop. Now you can check back again. Thanks 👍